README.rst

```diff
@@ -1,6 +1,6 @@
-====================================
- REANA demo example - "hello world"
-====================================
+==============================
```

Comment:
One more trailing '='.  Note that the first big heading "hangs over" by one character from left and right in ReST.